src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaFlowPathRepositoryTest.java

```diff
@@ -394,6 +394,23 @@ public void shouldFindFlowPathIdsByFlowIds() {
         assertTrue(pathIds.contains(flowB.getReversePathId()));
     }
 
+    @Test
+    public void findPathIdsBySharedBandwidthGroupId() {
+        String sharedBandwidthGroupId = "";
```

Comment:
Just interesting: why groupId is empty string?

Reply:
just because my memory sometimes fails 😆 
fixed